{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -389,7 +386,7 @@ def DATABASES(self):
     # https://sentry.io/for/django/
 
     try:
-        import sentry_sdk  # noqa
+        import sentry_sdk
```

Comment:
Sentry is often present but not always. Therefore, I suggest what follows
```suggestion
        import sentry_sdk  # noqa: F401
```